test/unit/account/states_test.rb

```diff
@@ -148,6 +148,63 @@ def test_suspend_master
     assert account.approved?
   end
 
+  test '.without_traffic_since_time_ago' do
+    accounts = FactoryGirl.create_list(:simple_account, 3)
+
+    # Account[0] with no traffic at all
+
+    # Account[1] with no traffic since TIME_TO_BE_INACTIVE ago
+    account = accounts[1]
+    cinstance = FactoryGirl.create(:cinstance, user_account: account)
+    cinstance.update_attribute(:first_daily_traffic_at, Account::States::TIME_TO_BE_INACTIVE.ago)
+
+    # Account[2] with traffic more recent than TIME_TO_BE_INACTIVE ago.
+    account = accounts[2]
+    cinstance = FactoryGirl.create(:cinstance, user_account: account)
+    cinstance.update_attribute(:first_daily_traffic_at, (Account::States::TIME_TO_BE_INACTIVE - 1.day).ago)
+
+    results = Account.without_traffic_since_time_ago.pluck(:id)
+    assert_includes     results, accounts[0].id
+    assert_includes     results, accounts[1].id
+    assert_not_includes results, accounts[2].id
+  end
+
+  test '.inactive_since_time_ago' do
+    accounts = FactoryGirl.create_list(:simple_account, 4)
+
+    # Account with no traffic since TIME_TO_BE_INACTIVE ago and was created before that
+    account = accounts[0]
```

Comment:
better to not use the same variable again `account` but:

`no_traffic_account = accounts[1]`

with variables like this these comment would be unnecessary.